PYTHON/AzureSQLDB/read_from_azure_sql_db.py

```diff
@@ -0,0 +1,30 @@
+# This sample allows you to read data from Azure SQL database 
+# Complete these steps before testing this funtion 
+#   1. Select Manage connections to connect to Azure SQL database 
+#   2. Copy the Alias name and replace it below
+
+@app.connection("sqlDB",alias="<alias for azure sql database>")
+@app.function("read_from_azure_sql_db")
+def read_from_azure_sql_db(sqlDB: udf.FabricSqlConnection)->str:
```

Comment:
similarly with udf.*, the templated currently have `fabric.functions.*`. We sound like we'll move to udf.* soon, so we could either wait until the train that updates the template gets to prod and then merge this PR, or we could merge this PR beforehand if we include the import `import fabric.functions as udf` at the top of each of these samples